ttnn/cpp/ttnn/operations/moreh/moreh_sum_backward/device/moreh_sum_backward_device_operation.hpp

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ttnn/decorators.hpp"
+#include "ttnn/device_operation.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+
+namespace ttnn::operations::moreh::moreh_sum_backward {
+struct MorehSumBackwardOperation {
+    struct operation_attributes_t {
+        const std::vector<int64_t> dims;
+        const bool keepdim;
+        const MemoryConfig input_grad_memory_config;
+        const DeviceComputeKernelConfig compute_kernel_config;
```

Comment:
I'm waiting for a response from the TT side on this part. Everything else looks good.

Reply:
TT mentioned that optional depends on the use-case, so it seems it doesn't need to be optional.